.github/workflows/ci.yml

```diff
@@ -110,17 +110,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Install latest nightly
-        run: |
-          rustup toolchain install nightly --component rustfmt
-          rustup override set nightly
-      - name: Formatting (miri, ui_test)
-        run: cargo fmt --all --check
-      - name: Formatting (everything else)
-        # TODO: Add `tests` (work in progress).
-        # Maybe change to `find . -name '*.rs'`, superseding the previous step.
-        run: find bench-cargo-miri benches cargo-miri test-cargo-miri -name '*.rs'
-          | xargs rustfmt --edition=2021 --config-path ./rustfmt.toml --check
+      - uses: dtolnay/install@rustup-toolchain-install-master
+      - run: |
+          rustup-toolchain-install-master -n miri -c rustfmt --no-default-components -- $(cat rust-version)
```

Comment:
I'm not a fan of duplicating this logic.

IMO this should keep installing the latest nightly, and then just run `./miri fmt --check`.
Or else we add it to the clippy job, but that would make bors fail on formatting errors which I wanted to avoid given how suboptimal rustfmt's formatting often is. But maybe it is time to stop my fight against windmills here...

Reply:
:+1: Changed to nightly